langchain4j/src/main/java/dev/langchain4j/agent/tool/DefaultToolExecutor.java

```diff
@@ -194,6 +204,13 @@ private static double getDoubleValue(
             String parameterName,
             Class<?> parameterType
     ) {
+        if (argument instanceof String) {
```

Comment:
This change is not required any more, right?

Reply:
I've put this one because some times llm respond with string instead of double when I was testing ollama tools.
Do you mean you fix it in another way ?

Reply:
I can move it to Ollama tools PR if you prefer ?

Reply:
Ah, I see. No, it is ok to keep it here. Perharps add a test?

Reply:
sure, done! it was tested before in my Ollama IT , but it's better to have it in the unit test.